mc/Mongo-Core.package/MongoIsMaster.class/instance/urlString.st

```diff
@@ -0,0 +1,5 @@
+replica set
+urlString
```

Comment:
How is this used? Can you think of a better name?

Reply:
The intent was to differenciate from the cases when a ZnUrl is useful vs this string version of the url that always comes in the responses. May be `host`?